### PR TITLE
adding Q&A from slack conversations

### PR DESCRIPTION
**Pull request by heckj**

### CODE REVIEW

File: docs/devguide/debugging_guide.rst [hunk 1]
```diff
@@ -76,3 +76,48 @@ Default discovery workflow
...
+    W 2016-01-29T21:12:43.783Z [on-tftp] [Tftp.Server] [Server] Tftp error
+     -> /lib/server.js:57
+    file:          monorail.ipxe
+    remoteAddress: 172.31.128.5
+    remotePort:    2070
+
+*Answer*:
+
+What I learned (so I may be wrong here, but think it’s accurate) is that when NIC’s a doing the bootloading/PXE
+process, they attempt to interact with TFTP in such a way that the first request almost always fails - it’s how
```
Comment (VulpesArtificem) [inline docs/devguide/debugging_guide.rst]: maybe: "...during the boot loading/PXE process the NICs will attempt to..."

File: docs/devguide/debugging_guide.rst [hunk 2]
```diff
@@ -76,3 +76,48 @@ Default discovery workflow
...
+    remotePort:    2070
+    W 2016-01-29T21:12:43.783Z [on-tftp] [Tftp.Server] [Server] Tftp error
+     -> /lib/server.js:57
+    file:          monorail.ipxe
+    remoteAddress: 172.31.128.5
+    remotePort:    2070
+
+*Answer*:
+
+What I learned (so I may be wrong here, but think it’s accurate) is that when NIC’s a doing the bootloading/PXE
```
Comment (VulpesArtificem) [inline docs/devguide/debugging_guide.rst]: NICs

**Comment by benbp** [comment]
Should we now implement a slack command called `/publishToDoc` that puts up a PR with the last 50 lines? :P

File: docs/devguide/debugging_guide.rst [hunk 3]
```diff
@@ -76,3 +76,48 @@ Default discovery workflow
...
+process, they attempt to interact with TFTP in such a way that the first request almost always fails - it’s how
+the C code in those nics are negotiating for talking with TFTP. So you’ll frequently see those errors in the logs,
```
Comment (VulpesArtificem) [inline docs/devguide/debugging_guide.rst]: "...is negotiating..."  also no comma after "in the logs"

File: docs/devguide/debugging_guide.rst [hunk 4]
```diff
@@ -76,3 +76,48 @@ Default discovery workflow
...
+
+When we're boostraping a node (or running a workflow against a node in general) with a NUC, we sometimes see these
+extended messages on the server's console reading `Link......  down`, and depending on the network configuration
+can see failures for the node to bootstrap and respond to PXE.
+
+*Answer*:
+
+The link down is a pernicious problem for PXE booting in general, and a part of the game that’s buried into how
+switches react and bring up and down ports. We’ve generally encouraged settings like “portfast” which more
+agressively bring up links that are going down and coming back up with a power cycle. In the NUC’s you’re using,
```
Comment (VulpesArtificem) [inline docs/devguide/debugging_guide.rst]: NUCs

File: docs/devguide/debugging_guide.rst [hunk 5]
```diff
@@ -76,3 +76,48 @@ Default discovery workflow
...
+you’ll see that extensively, but it happens on all networks. It you have spanning-tree enabled, some things
```
Comment (VulpesArtificem) [inline docs/devguide/debugging_guide.rst]: "If you have spanning-tree, or something like that..." ?

**Comment by heckj** [comment]
@VulpesArtificem @benbp updated!

**Comment by benbp** [comment]
:+1:

**Comment by VulpesArtificem** [comment]
:+1:

**Pull request merged**